src/components/form-elements/Textarea.jsx

```diff
@@ -38,11 +39,11 @@ const Textarea = props => {
     'sg-textarea--auto-height': autoHeight
   }, className);
 
-  return <textarea className={textareaClass} value={value} {...additionalProps} />;
+  return <Type className={textareaClass} value={value} {...additionalProps} />;
 };
 
 Textarea.propTypes = {
-  type: PropTypes.string,
+  Type: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```

Comment:
why capitalized and why func allowed?

Reply:
string = normal component
func = react class

capitalized take variable  if small create html tag